drivers/display/Kconfig.stm32_ltdc

```diff
@@ -37,3 +39,23 @@ config STM32_LTDC_RGB565
 	  (2 bytes per pixel)
 
 endchoice
+
+config STM32_LTDC_FB_NUM
+	int "Frame buffer number"
+	default 1
+	range 0 2
+	help
+	  STM32 LTDC frame buffer number config:
+	    - 0 frame buffer maintained by application, must write with full screen pixels.
+	    - 1 single frame buffer in stm32 ltdc driver.
+	    - 2 double frame buffer in stm32 ltdc driver.
+
+config STM32_LTDC_DISABLE_FMC_BANK1
+	bool "Disable FMC bank1 for STM32F7/H7 series"
+	depends on SOC_SERIES_STM32H7X || SOC_SERIES_STM32F7X
+	default y
+	help
+	  Disable FMC bank1 to avoid access conflict. More refer
+	  AN4861 4.6 Special recommendations for Cortex-M7 (STM32F7/H7).
```

Comment:
```suggestion
	  Disable FMC bank1 if not used to prevent speculative read accesses.
	  Refer to AN4861 "4.6 Special recommendations for Cortex-M7 (STM32F7/H7)".
```

Reply:
Done, thanks!